libs/translator/reader/read_light.cpp

```diff
@@ -296,6 +226,86 @@ AtNode *_ReadLightShaping(const UsdPrim &prim, UsdArnoldReaderContext &context)
 
 } // namespace
 
+
+void ReadLightCommon(const UsdPrim& prim, AtNode *node, const TimeSettings &time)
+{
+#if PXR_VERSION >= 2111
+    UsdLuxLightAPI light(prim);
+#else
+    UsdLuxLight light(prim);
+#endif
+    
+    // This function is computing intensity, color, and eventually color
+    // temperature. Another solution could be to export separately these
+    // parameters, but it seems simpler to do this for now
+
+    VtValue colorValue;
+    GfVec3f color(1.f, 1.f, 1.f);
+    if (GET_LIGHT_ATTR(light, Color).Get(&colorValue, time.frame))
+        color = colorValue.Get<GfVec3f>();
+
+    VtValue intensityValue;
+    float intensity = 1.f;
+    if (GET_LIGHT_ATTR(light, Intensity).Get(&intensityValue, time.frame)) {
+        intensity = VtValueGetFloat(intensityValue);
+        AiNodeSetFlt(node, str::intensity, intensity);
+    }
+
+    VtValue exposureValue;
+    float exposure = 0.f;
+    if (GET_LIGHT_ATTR(light, Exposure).Get(&exposureValue, time.frame)) {
+        exposure = VtValueGetFloat(exposureValue);
+        AiNodeSetFlt(node, str::exposure, exposure);
+    }
+
+    VtValue enableTemperatureValue;
+    if (GET_LIGHT_ATTR(light, EnableColorTemperature).Get(&enableTemperatureValue, time.frame)) {
+        float colorTemp = 6500;
+        if (VtValueGetBool(enableTemperatureValue) &&
+            GET_LIGHT_ATTR(light, ColorTemperature).Get(&colorTemp, time.frame)) {
+            color = GfCompMult(color, UsdLuxBlackbodyTemperatureAsRgb(colorTemp));
+        }
+    }
+    AiNodeSetRGB(node, str::color, color[0], color[1], color[2]);
+
+    VtValue diffuseValue;
+    if (GET_LIGHT_ATTR(light, Diffuse).Get(&diffuseValue, time.frame)) {
+        AiNodeSetFlt(node, str::diffuse, VtValueGetFloat(diffuseValue));
+    }
+    VtValue specularValue;
+    if (GET_LIGHT_ATTR(light, Specular).Get(&specularValue, time.frame)) {
+        AiNodeSetFlt(node, str::specular, VtValueGetFloat(specularValue));
+    }
+
+    /*
+    This is preventing distant lights from working properly, so we should only
+    do it where it makes sense VtValue normalizeAttr.
+    if(light.GetNormalizeAttr().Get(&normalizeAttr, time.frame))
+       AiNodeSetBool(node, "normalize", normalizeAttr.Get<bool>());
+    */
+    // Normalize attribute only for meshlights
+    if (AiNodeEntryGetNameAtString(AiNodeGetNodeEntry(node)) == str::mesh_light) {
+        VtValue normalize;
+        if (GET_LIGHT_ATTR(light, Normalize).Get(&normalize, time.frame)) {
+            AiNodeSetBool(node, str::normalize, VtValueGetBool(normalize));
```

Comment:
to be consistent with the existing code, this normalize attribute should be set from the mesh translator just before or after it calls `ReadLightCommon`. This way we're sure it's only set in that case and not e.g. if the mesh_light is created from another translator, like GeometryLightTranslator